content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -10,33 +10,29 @@ weight: 10
 
 {{% capture overview %}}
 
-ReplicaSet is the next-generation Replication Controller. The only difference
-between a _ReplicaSet_ and a
-[_Replication Controller_](/docs/concepts/workloads/controllers/replicationcontroller/) right now is
-the selector support. ReplicaSet supports the new set-based selector requirements
-as described in the [labels user guide](/docs/concepts/overview/working-with-objects/labels/#label-selectors)
-whereas a Replication Controller only supports equality-based selector requirements.
+A ReplicaSet's purpose is to maintain a stable set of replica Pods running at any given time. As such, it is often
+used to guarantee the availability of a specified number of identical pods.
+
 
 {{% /capture %}}
 
 {{% capture body %}}
 
-## How to use a ReplicaSet
+## How a ReplicaSet Works
+
+A ReplicaSet is defined with fields including, a selector which specifies how to identify pods it can acquire, a number
+of replicas indicating how many pods it should be maintaining, and a pod template specifying the data of new pods
+it should create to meet the number of replicas criteria. A ReplicaSet then fulfills its purpose by creating
+and deleting Pods as needed to reach the desired number, using its given pod template for the creation.
 
-Most [`kubectl`](/docs/user-guide/kubectl/) commands that support
-Replication Controllers also support ReplicaSets. One exception is the
-[`rolling-update`](/docs/reference/generated/kubectl/kubectl-commands#rolling-update) command. If
-you want the rolling update functionality please consider using Deployments
-instead. Also, the
-[`rolling-update`](/docs/reference/generated/kubectl/kubectl-commands#rolling-update) command is
-imperative whereas Deployments are declarative, so we recommend using Deployments
-through the [`rollout`](/docs/reference/generated/kubectl/kubectl-commands#rollout) command.
+The link a ReplicaSet has to its Pods is via the [metadata.ownerReferences][/docs/concepts/workloads/controllers/garbage-collection/#owners-and-dependents]
```

Comment:
Suggestion: "is via the Pod's metadata.ownerReferences field." That way the reader won't jump to the incorrect conclusion that we're talking about the ReplicaSet's metadata.ownerReferences field.